sw/device/meson.build

```diff
@@ -11,7 +11,7 @@ subdir('exts')
 # These definitions should only be available to directories which define executables.
 make_embedded_target_outputs = ['@BASENAME@.bin', '@BASENAME@.dis', '@BASENAME@.vmem']
 make_embedded_target = [
-  prog_python, meson.source_root() + '/util/embedded_target.py',
+  prog_python, meson.source_root() / 'util/embedded_target.py',
```

Comment:
The fact that this works is almost worrying.

Reply:
Meson's docummentation explicitly says somewhere "use / to join path components because platform-independent!" so I'm not going to even question it.